client/src/pages/reviews/UpdateReviewForm.tsx

```diff
@@ -45,6 +45,7 @@ function UpdateReviewForm({
     mutationFn: updateReview,
     onSuccess: () => {
       queryClient.invalidateQueries({ queryKey: ['reviews', book.id] });
+      queryClient.invalidateQueries({ queryKey: ['books', book.id] });
```

Comment:
To jest na pewno potrzebne?

Reply:
no tak, bez tego average rating ksiazki Ci sie zaktualizuje dopiero jak zrefreshujesz strone.

Reply:
Fakt, racja

Reply:
Chociaz może fajnie by było nie robić całego refetcha. Zamiast tego w tym requescie do update czy ram dodania zwracać końcowy avarageRating i podstawić go do danych na stronie. Te całe query mają takie opcje chyba

Reply:
jak ci się nie chce to ja mogę sprobowac to ogarnąć, bo ten refetch to dużo danych pobiera, a w sumie chcemy jedno pole zaktualizować

Reply:
no niby ta ale czy zrefetchowanie calej ksiazki jeszcze raz a samego ratingu az taka roznice nam tu zrobi